pkg/kubelet/volumemanager/cache/actual_state_of_world_test.go

```diff
@@ -590,7 +590,7 @@ func Test_MarkDeviceAsMounted_Positive_NewVolume(t *testing.T) {
 	}
 
 	// Act
-	err = asw.MarkDeviceAsMounted(generatedVolumeName, devicePath, deviceMountPath)
+	err = asw.MarkDeviceAsMounted(generatedVolumeName, devicePath, deviceMountPath, "")
```

Comment:
Can you add a unit test that checks the selinuxcontext is set?

Reply:
Added a few tests.